docs/monitoring/faq/webhooks.mdx

```diff
@@ -77,3 +77,116 @@ When delivering events to your webhook endpoint we follow these guidelines
 - If your endpoint returns a 5xx status code in less than 5 seconds we retry up to 2 times with exponential backoff.
 - If your endpoint returns a 4xx status code, we declare the delivery failed and do not retry.
 - If your endpoint returns any body do not read it
+
+## Example with Modal
+
+### Setup
+
+For an example of how to set this up, we will use [Modal](https://modal.com/).
+
+First, create a Modal account. Then, locally install the Modal SDK:
+
+```shell
+pip install modal
+```
+
+### Secrets
+
+Next, you will need to set up some secrets.
+
+First, we will set up a secret to use to authenticate to our server.
+You can see instructions for secrets [here](https://modal.com/docs/guide/secrets).
+For this purpose, let's call our secret `ls-webhook` and have it set an environment variable with the name `LS_WEBHOOK`.
```

Comment:
I found the explanation of the secrets a bit confusing in the text above. It wasn't obvious to me when I read the docs last week that you mean "put a secret in the query parameters" -- I expected to find some kind of option for secrets inside the LangSmith UI/SDK.